app/modelEvents.php

```diff
@@ -1,7 +1,7 @@
 <?php
 
 // UUID-based models
-$models = ['Talk', 'Conference', 'TalkVersion', 'TalkVersionRevision'];
+$models = ['Talk', 'Conference', 'TalkVersion', 'TalkVersionRevision', 'Bio'];
 
 foreach ($models as $model) {
     $model::creating(function ($model) {
```

Comment:
Not new to this PR, but I think it makes more sense to put the UUID generation in a `save` override on the `UuidBase` model like we ended up doing on other projects, especially since we have that base model anyways. This separate-file-model-event-binding stuff was really just a workaround for weird PHPUnit static state crap anyways.